crates/fluvio-socket/src/multiplexing.rs

```diff
@@ -459,27 +482,31 @@ impl MultiPlexingResponseDispatcher {
                 }
                 SharedSender::Queue(queue_sender) => {
                     trace!("found stream");
-                    queue_sender.send(Some(msg)).await.map_err(|err| {
-                        IoError::new(
-                            ErrorKind::BrokenPipe,
-                            format!(
-                                "problem sending to queue socket: {}, err: {}",
-                                correlation_id, err
-                            ),
-                        )
-                        .into()
-                    })
+                    // sender was dropped before response arrives
+                    if queue_sender.is_closed() {
+                        debug!("attempt to send data to closed socket: {}", correlation_id);
```

Comment:
Can you use trace format? 
`debug!(correlation_id,"attemp....")`

Reply:
Done